### DISCUSSION TITLE
Discord Channel URL

### DISCUSSION THREAD

**Discussion by ragokan** [remix-run/remix#4193] [General]
Hello, I would like to ask a question about the Discord Channel URL. I got it from the https://remix.run/ website but sadly it is not working. I tried to connect at Twitter but sadly I was not able to. So, may I have the URL to Discord channel if it exists?

**Comment by nullndr** [comment]
Here https://discord.com/channels/770287896669978684/771069639479394315

**Comment by j314h** [comment]
Hello, the link does not work (for me anyway), and is it possible that there is a problem?

**Reply by ragokan** [reply]
It did not work for me, too. I searched as Remix in Discord and found it. You can try this link, too
https://discord.gg/Mus3zE7d

**Reply by j314h** [reply]
Thanks a lot

**Comment by saibotrellum** [comment]
same here, link from website throws Discord error. (Invite invalid. May be expired or you don't have permissions.)
https://discord.gg/Mus3zE7d worked.
Can't find the website-repo, so no PR, sorry

**Reply by j314h** [reply]
it worked for me it must be outdated

**Reply by j314h** [reply]
try this
[https://discord.gg/Y5hVm4HG](https://discord.gg/Y5hVm4HG)
